{{project_name}}/.pre-commit-config.yaml.jinja

```diff
@@ -15,6 +15,8 @@ repos:
     hooks:
       - id: prettier
         types: [yaml]
+         endOfLine: auto
+        proseWrap: "always"
```

Comment:
This should be in a new `.prettierrc` folder within the template

Reply:
Can you explain why it should be there when it seems to work here?

Reply:
I don't think this will work. The indentation is off, and this is an option for `prettier`, not for `pre-commit`